meta/src/store/state_machine.rs

```diff
@@ -657,7 +661,7 @@ impl StateMachine {
         }
     }
 
-    fn process_add_date_node(&self, cluster: &str, node: &NodeInfo) -> CommandResp {
+    fn common_function(&self, cluster: &str, node: &NodeInfo) -> CommandResp {
```

Comment:
I don't see any semantics attached to the function name, and I don't see any comments, I can't understand what this function does